### ISSUE TITLE
Settings global exclude pattern in my $HOME/.rubocop.yml

### ISSUE THREAD

**Issue by zw963** [bbatsov/rubocop#2261]
I want to add all my project exclude pattern in my $HOME/.rubycop.yml

so, I have to add like this:
```yml
AllCops:
  Include:
    - '**/Rakefile'
    - '**/config.ru'
  Exclude:
  - '**/vendor/**/*'
  - '**/spec/fixtures/**/*'
  - '**/db/schema.rb'
  - '**/db/migrate/*'
  - '**/bin/bundle'
  - '**/bin/rails'
  - '**/bin/rake'
  - '**/bin/setup'
  - '**/bin/spring'
  - '**/config/**/*'
  - '**/script/**/*'
```

But, I think it maybe not reasonable, if remove the leading ``** `` is more reasonable.
no one hope exclude some directory in $HOME after all.

Thanks

**Comment by jonas054** [comment]
This problem is very similar to the one that was fixed in #2264. Relative paths in files named `.rubocop.yml` are relative to the directory where that file is. The file `$HOME/.rubocop.yml` should be an exception to this rule, and paths there should be relative to the current directory.

**Comment by zw963** [comment]
Thank for reply.
I think `$HOME/.rubocop.yml`, it should be the `default value` for current project
missing config, so, it should be same behavior.
